engine/src/main/java/org/terasology/rendering/dag/nodes/AmbientOcclusionPassesNode.java

```diff
@@ -75,9 +79,9 @@ public void process() {
         if (renderingConfig.isSsao()) {
             PerformanceMonitor.startActivity("rendering/ambientOcclusionPasses");
             // TODO: consider moving these into initialise without breaking existing implementation
-            sceneOpaque = frameBuffersManager.getFBO("sceneOpaque");
-            ssaoBlurredFBO = frameBuffersManager.getFBO("ssaoBlurred");
-            ssaoFBO = frameBuffersManager.getFBO("ssao");
+            sceneOpaque = dynamicFBM.getFBO(DefaultDynamicFBOs.ReadOnlyGBuffer.getResourceUrn());
```

Comment:
- Just checked if it's possible. And yes, it can be done.
- Will do.
- Actually requireFBO increases the usage count, and answer for your second question will be no, it should be not possible.